bindings/python/sir.i

```diff
@@ -5,8 +5,13 @@
  * Copyright: Copyright (c) 2018-2023
  * Version:   2.2.4
  * License:   The MIT License (MIT)
+ *
+ * -----------------------------------------------------------------------------
+ *
  * SPDX-License-Identifier: MIT
  *
+ * Copyright (c) 2018-2023 Ryan M. Lederman <lederman@gmail.com>
```

Comment:
I don't want to have these duplicate copyright sections, whether this gives us "stricter" compliance or not... if there isn't a clean way to consolidate I would rather not change any file that already has copyright info either inline or in the .deps5

Reply:
Changed the syntax to avoid that …